git/script/makeReleaseNotes.ts

```diff
@@ -59,7 +60,7 @@ async function commits(
       commits.push({ hash, author, date, ...conventionalCommit });
     } catch {
       log(`  Ignoring unconventional commit  ${hash}`);
-      log(`    ${message.substring(0, terminalWidth - 7)}...`);
+      log(`    ${elideEnd(message, { maxLength: terminalWidth - 4 })}`);
```

Comment:
`7` was used to account for the 4 spaces of indentation plus the 3 periods at the end.

Now `4` is used to account for the indentation and `elideEnd()` handles accommodating the ellipses.